fsh-generated/resources/ActorDefinition-au-erequesting-actor-patient.json

```diff
@@ -6,6 +6,7 @@
   "title": "AU eRequesting Patient",
```

Comment:
Commenting here to get an inline comment. 
fsh-generated files are not tracked anymore. Please remove from this PR.

Reply:
I can see the deleted file but fsh-generated and the deleted file are still showing in this PR which makes me think that fsh-generated wasn't properly untracked. If it was, fsh-generated and files inside it, whether they are new, modified or deleted, should not be shown in the PR.

Reply:
Still showing in PR.

Reply:
done